lib/chef/knife/azurerm_base.rb

```diff
@@ -27,6 +27,10 @@ class Chef
   class Knife
     module AzurermBase
 
+      ## azure-xplat-cli versio that introduced deprecation of Windows Credentials
+      ## Manager (WCM) usage for authentication credentials storage purpose ##
+      XPLAT_VERSION_WITH_WCM_DEPRECATED ||= "0.10.5"
```

Comment:
This is a constant, so you can just assign the value here rather than using `||=`. As a constant being set in the class rather than in a method, this won't be seen twice.